nautobot/core/views/mixins.py

```diff
@@ -1082,94 +1109,14 @@ def perform_bulk_create(self, request):
             return self.form_invalid(form)
 
 
-class ObjectBulkUpdateViewMixin(NautobotViewSetMixin, BulkUpdateModelMixin, EditAndDeleteAllModelMixin):
+class ObjectBulkUpdateViewMixin(NautobotViewSetMixin, BulkUpdateModelMixin, EditAndDeleteModelMixin):
     """
     UI mixin to bulk update model instances.
     """
 
     filterset_class = None
     bulk_update_form_class = None
 
-    def _process_bulk_update_form(self, form):
```

Comment:
You've removed this method but I don't see where you've removed any code that was previously calling it?

Reply:
This is the part of UIViewSet that was moved to Bulk Edit System Job

Reply:
My point is that presumably *something* was calling `_process_bulk_update_form()` previously, but I don't see where that "something" was removed from the code base. You've removed the definition of the method but not the code that was calling that method?